tests/components/scorecard_chart.test.ts

```diff
@@ -281,6 +283,28 @@ describe("Scorecard charts", () => {
     }
   });
 
+  test.each([{ keyValue: "A3" }, { baseline: "A3" }])(
+    "Scorecard elements take the bold/italic style into account when finding the best font size",
+    async (scorecardArg) => {
+      const string = "This is a long string that will be the keyvalue";
+      let ctxFontOFMeasuredValue = "";
+      jest
+        .spyOn(MockCanvasRenderingContext2D.prototype, "measureText")
+        .mockImplementation(function (this: MockCanvasRenderingContext2D, text) {
```

Comment:
you have to restore the mock for the following tests that use it. I think you either the mock is incomplete and fails to fallback on the original behaviour or the wait you implement it is correct and it should be cleared after the test with [this](https://jestjs.io/docs/mock-function-api#mockfnmockrestore)

Reply:
Added a line to clean the mock :+1: 

I think it's better for the mock to return a random value rather than fallback to the original behaviour, so we can catch when we fail to clean the mock (like in this task)